engines/registry/app/models/access_scope.rb

```diff
@@ -88,19 +88,19 @@ def allowed_paths(system = nil)
     allowed_products = (active_products & access_policies_yml.keys)
     allowed_glob_paths = access_policies_yml.values_at(*allowed_products).flatten
 
-    if system && system.hybrid?
-      allowed_product_class = allowed_products.detect { |s| s.downcase.include?('ltss') }
-      if allowed_product_class.present?
-        activation_state = SccProxy.scc_check_subscription_expiration(
-          {}, system.login, system.system_token, Rails.logger, system.proxy_byos_mode, allowed_product_class
+    if system && system.hybrid? && allowed_products.any? { |s| !Product.find_by(product_class: s).free? }.present?
+      auth_header = {
+        Authorization: ActionController::HttpAuthentication::Basic.encode_credentials(system.login, system.password)
+      }
+      activation_state = SccProxy.scc_check_subscription_expiration(
+        auth_header, system.login, system.system_token, Rails.logger, system.proxy_byos_mode, allowed_products.first
```

Comment:
It feels random to use the `.first` product here. Are we sure that the first product is actually the product that was activated with SCC?

Reply:
we have a product class, one product class

in `allowed_product_classes` (which should be `allowed_product_class` will be always one product class, correct ?
like access policy has `- FOO: /a/b/c/**` and the paths can be 18 but `FOO` is one, yes it comes in an array but it is an array of size 1

Reply:
Update the approach, to remove the non active non free extensions from the allowed paths